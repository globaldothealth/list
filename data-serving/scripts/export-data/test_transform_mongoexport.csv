_id,caseReference.additionalSources,caseReference.sourceEntryId,caseReference.sourceId,caseReference.sourceUrl,caseReference.uploadIds,caseReference.verificationStatus,demographics.ageRange.end,demographics.ageRange.start,demographics.ethnicity,demographics.gender,demographics.nationalities,demographics.occupation,events,genomeSequences,location.administrativeAreaLevel1,location.administrativeAreaLevel2,location.administrativeAreaLevel3,location.country,location.geoResolution,location.geometry.latitude,location.geometry.longitude,location.name,location.place,location.query,notes,pathogens,preexistingConditions.hasPreexistingConditions,preexistingConditions.values,revisionMetadata.creationMetadata.curator,revisionMetadata.creationMetadata.date,revisionMetadata.creationMetadata.notes,revisionMetadata.editMetadata.curator,revisionMetadata.editMetadata.date,revisionMetadata.editMetadata.notes,revisionMetadata.revisionNumber,SGTF,symptoms.status,symptoms.values,transmission.linkedCaseIds,transmission.places,transmission.routes,travelHistory.travel.dateRange.end,travelHistory.travel.dateRange.start,travelHistory.travel.location.name,travelHistory.travel.methods,travelHistory.travel.purpose,travelHistory.traveledPrior30Days,vaccines.0.name,vaccines.0.batch,vaccines.0.date,vaccines.0.sideEffects,vaccines.1.name,vaccines.1.batch,vaccines.1.date,vaccines.1.sideEffects,vaccines.2.name,vaccines.2.batch,vaccines.2.date,vaccines.2.sideEffects,vaccines.3.name,vaccines.3.batch,vaccines.3.date,vaccines.3.sideEffects
ObjectId(6817283abaa89324a90109aa),[],,787123878aa90909811aaff1,http://foo/bar.csv,"[""bb12399abbb19230900aa123""]",UNVERIFIED,69,60,,Male,[],,"[{""name"":""confirmed"",""dateRange"":{""start"":{""$date"":""2021-10-01T00:00:00.000Z""},""end"":{""$date"":""2021-01-01T00:00:00.000Z""}}}]",[],,,,Antarctica,Country,-79.402,0.323,Antarctica,,,,[],,,ingestion@example.com,2021-01-02T13:42:34.991Z,,,,,0,,,,,,,,,,,true,,,,,,,,,,,,,,,,
ObjectId(798989a98998acc98989a1bb),[],,787123878aa90909811aaff1,http://foo/bar.csv,"[""bb12399abbb19230900aa123""]",UNVERIFIED,29,20,,Female,[],,"[{""name"":""confirmed"",""dateRange"":{""start"":{""$date"":""2021-01-05T00:00:00.000Z""},""end"":{""$date"":""2021-01-05T00:00:00.000Z""}}}]",[],,,,Antarctica,Country,-79.402,0.323,Antarctica,,,,[],,,ingestion@example.com,2021-01-02T13:42:34.991Z,,,,,0,,,,,,,,,,,true,,,,,,,,,,,,,,,,
