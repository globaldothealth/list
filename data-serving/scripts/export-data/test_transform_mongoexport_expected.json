[
  {"SGTF": "NA", "_id": "ObjectId(6817283abaa89324a90109aa)", "caseReference.additionalSources": null, "caseReference.sourceEntryId": "", "caseReference.sourceId": "787123878aa90909811aaff1", "caseReference.sourceUrl": "http://foo/bar.csv", "caseReference.uploadIds": "bb12399abbb19230900aa123", "caseReference.verificationStatus": "UNVERIFIED", "demographics.ageRange.end": "69", "demographics.ageRange.start": "60", "demographics.ethnicity": "", "demographics.gender": "Male", "demographics.nationalities": null, "demographics.occupation": "", "events.confirmed.date": "2021-01-01", "events.confirmed.value": null, "events.firstClinicalConsultation.date": "", "events.hospitalAdmission.date": "", "events.hospitalAdmission.value": "", "events.icuAdmission.date": "", "events.icuAdmission.value": "", "events.onsetSymptoms.date": "", "events.outcome.date": "", "events.outcome.value": "", "events.selfIsolation.date": "", "genomeSequences": "[]", "genomeSequences.repositoryUrl": "", "genomeSequences.sampleCollectionDate": "", "genomeSequences.sequenceId": "", "genomeSequences.sequenceLength": "", "genomeSequences.sequenceName": "", "location.administrativeAreaLevel1": "", "location.administrativeAreaLevel2": "", "location.administrativeAreaLevel3": "", "location.country": "Antarctica", "location.geoResolution": "Country", "location.geometry.latitude": "-79.402", "location.geometry.longitude": "0.323", "location.name": "Antarctica", "location.place": "", "notes": "", "pathogens": null, "preexistingConditions.hasPreexistingConditions": "", "preexistingConditions.values": "", "revisionMetadata.creationMetadata.date": "2021-01-02T13:42:34.991Z", "revisionMetadata.creationMetadata.notes": "", "revisionMetadata.editMetadata.date": "", "revisionMetadata.editMetadata.notes": "", "revisionMetadata.revisionNumber": "0", "symptoms.status": "", "symptoms.values": "", "transmission.linkedCaseIds": "", "transmission.places": "", "transmission.routes": "", "travelHistory.travel.dateRange.end": "", "travelHistory.travel.dateRange.start": "", "travelHistory.travel.location.administrativeAreaLevel1": "", "travelHistory.travel.location.administrativeAreaLevel2": "", "travelHistory.travel.location.administrativeAreaLevel3": "", "travelHistory.travel.location.country": "", "travelHistory.travel.location.geoResolution": "", "travelHistory.travel.location.geometry.coordinates": "", "travelHistory.travel.location.name": "", "travelHistory.travel.location.place": "", "travelHistory.travel.methods": "", "travelHistory.travel.purpose": "true", "travelHistory.traveledPrior30Days": "", "vaccines.0.batch": "", "vaccines.0.date": "", "vaccines.0.name": "", "vaccines.0.sideEffects": "", "vaccines.1.batch": "", "vaccines.1.date": "", "vaccines.1.name": "", "vaccines.1.sideEffects": "", "vaccines.2.batch": "", "vaccines.2.date": "", "vaccines.2.name": "", "vaccines.2.sideEffects": "", "vaccines.3.batch": "", "vaccines.3.date": "", "vaccines.3.name": "", "vaccines.3.sideEffects": null, "variantOfConcern": ""}
, {"SGTF": "NA", "_id": "ObjectId(798989a98998acc98989a1bb)", "caseReference.additionalSources": null, "caseReference.sourceEntryId": "", "caseReference.sourceId": "787123878aa90909811aaff1", "caseReference.sourceUrl": "http://foo/bar.csv", "caseReference.uploadIds": "bb12399abbb19230900aa123", "caseReference.verificationStatus": "UNVERIFIED", "demographics.ageRange.end": "29", "demographics.ageRange.start": "20", "demographics.ethnicity": "", "demographics.gender": "Female", "demographics.nationalities": null, "demographics.occupation": "", "events.confirmed.date": "2021-01-05", "events.confirmed.value": null, "events.firstClinicalConsultation.date": "", "events.hospitalAdmission.date": "", "events.hospitalAdmission.value": "", "events.icuAdmission.date": "", "events.icuAdmission.value": "", "events.onsetSymptoms.date": "", "events.outcome.date": "", "events.outcome.value": "", "events.selfIsolation.date": "", "genomeSequences": "[]", "genomeSequences.repositoryUrl": "", "genomeSequences.sampleCollectionDate": "", "genomeSequences.sequenceId": "", "genomeSequences.sequenceLength": "", "genomeSequences.sequenceName": "", "location.administrativeAreaLevel1": "", "location.administrativeAreaLevel2": "", "location.administrativeAreaLevel3": "", "location.country": "Antarctica", "location.geoResolution": "Country", "location.geometry.latitude": "-79.402", "location.geometry.longitude": "0.323", "location.name": "Antarctica", "location.place": "", "notes": "", "pathogens": null, "preexistingConditions.hasPreexistingConditions": "", "preexistingConditions.values": "", "revisionMetadata.creationMetadata.date": "2021-01-02T13:42:34.991Z", "revisionMetadata.creationMetadata.notes": "", "revisionMetadata.editMetadata.date": "", "revisionMetadata.editMetadata.notes": "", "revisionMetadata.revisionNumber": "0", "symptoms.status": "", "symptoms.values": "", "transmission.linkedCaseIds": "", "transmission.places": "", "transmission.routes": "", "travelHistory.travel.dateRange.end": "", "travelHistory.travel.dateRange.start": "", "travelHistory.travel.location.administrativeAreaLevel1": "", "travelHistory.travel.location.administrativeAreaLevel2": "", "travelHistory.travel.location.administrativeAreaLevel3": "", "travelHistory.travel.location.country": "", "travelHistory.travel.location.geoResolution": "", "travelHistory.travel.location.geometry.coordinates": "", "travelHistory.travel.location.name": "", "travelHistory.travel.location.place": "", "travelHistory.travel.methods": "", "travelHistory.travel.purpose": "true", "travelHistory.traveledPrior30Days": "", "vaccines.0.batch": "", "vaccines.0.date": "", "vaccines.0.name": "", "vaccines.0.sideEffects": "", "vaccines.1.batch": "", "vaccines.1.date": "", "vaccines.1.name": "", "vaccines.1.sideEffects": "", "vaccines.2.batch": "", "vaccines.2.date": "", "vaccines.2.name": "", "vaccines.2.sideEffects": "", "vaccines.3.batch": "", "vaccines.3.date": "", "vaccines.3.name": "", "vaccines.3.sideEffects": null, "variantOfConcern": ""}
]
